keel-api/src/test/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigControllerTests.kt

```diff
@@ -184,7 +189,18 @@ internal class DeliveryConfigControllerTests : JUnit5Minutests {
         |      "name": "prod",
         |      "constraints": [
         |        {
+        |          "type": "depends-on",
         |          "environment": "test"
+        |        },
+        |        {
+        |          "type": "time-window",
+        |          "windows": [
+        |            {
+        |              "hours": "6-18",
+        |              "days": "Monday-Friday",
+        |              "tz": "America/Los_Angeles"
```

Comment:
Should the YAML payload have the `tz` field as well?

Reply:
Yeah, we discussed this previously. It's necessary otherwise the server is going to interpret those hours as UTC

Reply:
No, it's an optional field.

Reply:
Also note the dynamic config to set a default TZ for windows other than UTC. This will be set to "America/Los_Angeles " so our users will only have to specify a TZ if they want something else.